README.md

```diff
@@ -839,20 +839,47 @@ export default function Top() {
 
 ### Cloudflare Bindings
 
-If you want to use Cloudflare's Bindings in your development environment, create `wrangler.toml` and configure it properly.
-
-```toml
-name = "my-project-name"
-compatibility_date = "2024-04-01"
-compatibility_flags = [ "nodejs_compat" ]
-pages_build_output_dir = "./dist"
+If you want to use Cloudflare's Bindings in your development environment, create `wrangler.jsonc` and configure it properly.
 
-# [vars]
-# MY_VARIABLE = "production_value"
-
-# [[kv_namespaces]]
-# binding = "MY_KV_NAMESPACE"
-# id = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+```jsonc
+// wrangler.jsonc
+{
+  "$schema": "node_modules/wrangler/config-schema.json",
+  "name": "my-project-name",
+  "main": "./dist/index.js",
+  "compatibility_date": "2025-08-03",
+  "compatibility_flags": [
+    "nodejs_compat"
+  ],
+  "assets": {
+    "directory": "./dist"
+  },
+  "vars": {
```

Comment:
Ahh, sorry! My bad. I wanted to say these settings for bindings are not necessary. So the JSON will be:

```jsonc
// wrangler.jsonc
{
  "$schema": "node_modules/wrangler/config-schema.json",
  "name": "my-project-name",
  "main": "./dist/index.js",
  "compatibility_date": "2025-08-03",
  "compatibility_flags": [
    "nodejs_compat"
  ],
  "assets": {
    "directory": "./dist"
  }
}
```